components/edit-collective/VirtualCard.js

```diff
@@ -311,15 +317,15 @@ const VirtualCard = props => {
         alignItems="center"
         shrink={0}
       >
-        {props.hasActions && (
+        {props.canEditVirtualCard && (
           <ActionsButton
             id={props.id}
             state={props.data.state}
             onSuccess={props.onSuccess}
             editHandler={props.editHandler}
+            deleteHandler={props.deleteHandler}
           />
         )}
-        <React.Fragment>{props.provider}</React.Fragment>
```

Comment:
What do we do about the provider then?

Reply:
I think there is space to put it before "Assigned on 12/16/21" like "Stripe. Assigned on 12/16/21."

Reply:
Still only for the Host admin, not for the Collective.

Reply:
Sadly no, it adds a line if I put it before "assigned"